.github/workflows/python-poetry-ci.yml

```
@@ -0,0 +1,41 @@
+name: Linting
+on:
+  workflow_call:
+
+
+jobs:
+  style:
+    runs-on: ubuntu-latest
+
+    name: Style checks
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup Poetry
+        uses: matrix-org/setup-python-poetry@v1
+
+      - name: Import ordering
```

This (import) reads a little bit like a verb, but changing it is probably even more clunky, so whatever, I guess we'll all know what it means soon enough.
This is making me wonder if it's 'order' or 'ordering'... hmm English. I would have said 'order' but you might know that I'm wrong!

```suggestion
      - name: Import order (isort)
```